new-log-viewer/src/services/LogExportManager.ts

```diff
@@ -0,0 +1,42 @@
+import downloadDecompressedLogs from "../utils/downloadDecompressedLogs";
+
+
+class LogExportManager {
+    private blob: Blob;
+
+    private numChunks: number;
+
+    constructor (numChunks?: number) {
```

Comment:
Instead of taking in `numChunks`, we can take in `numEvents` and do the chunk size calculation here?

Reply:
I still think it is a good practice to pass the parameters directly to constructors, instead of making calculations inside. lmk your thoughts